src/Automation/Automation/ChangeLog.md

```diff
@@ -19,6 +19,9 @@
 -->
 ## Upcoming Release
 
+## Version 1.9.2
```

Comment:
Please do NOT create a new version tag in the ChangeLog.md. You can put it under ## Upcoming Release section.

Reply:
updated